src/dlgIRC.cpp

```diff
@@ -40,20 +40,19 @@
 #include "post_guard.h"
 
 
-QString dlgIRC::HostNameCfgItem = QStringLiteral("irc_host");
-QString dlgIRC::HostPortCfgItem = QStringLiteral("irc_port");
-QString dlgIRC::NickNameCfgItem = QStringLiteral("irc_nick");
-QString dlgIRC::ChannelsCfgItem = QStringLiteral("irc_channels");
-QString dlgIRC::DefaultHostName = QStringLiteral("irc.freenode.net");
+QString dlgIRC::HostNameCfgItem = QLatin1String("irc_host");
+QString dlgIRC::HostPortCfgItem = QLatin1String("irc_port");
+QString dlgIRC::NickNameCfgItem = QLatin1String("irc_nick");
+QString dlgIRC::ChannelsCfgItem = QLatin1String("irc_channels");
+QString dlgIRC::DefaultHostName = QLatin1String("irc.freenode.net");
 int dlgIRC::DefaultHostPort = 6667;
-QString dlgIRC::DefaultNickName = QStringLiteral("Mudlet");
-QStringList dlgIRC::DefaultChannels = QStringList() << QStringLiteral("#mudlet");
+QString dlgIRC::DefaultNickName = QLatin1String("Mudlet");
+QStringList dlgIRC::DefaultChannels = QStringList() << QLatin1String("#mudlet");
 
 dlgIRC::dlgIRC(Host* pHost) : mpHost(pHost), mInputHistoryMax(8), mIrcStarted(false), mReadyForSending(false), mConnectedHostName()
 {
     setupUi(this);
-    setWindowTitle(tr("%1 - Mudlet IRC Client").arg(mpHost->getName()));
-    setWindowIcon(QIcon(QStringLiteral(":/icons/mudlet_irc.png")));
+    setWindowIcon(QIcon(QLatin1String(":/icons/mudlet_irc.png")));
```

Comment:
It'll work because it'll create a new QString() at runtime instead, basically defeating the efficiency.